ui/src/actions/beamline.js

```diff
@@ -4,12 +4,6 @@ import {
   sendPrepareBeamlineForNewSample,
 } from '../api/beamline';
 import { sendLogFrontEndTraceBack } from '../api/log';
-// The different states a beamline attribute can assume.
-export const STATE = {
-  IDLE: 'READY',
-  BUSY: 'BUSY',
-  ABORT: 'UNUSABLE',
-};
```

Comment:
Moved to `reducer/beamline` since that's the only place it's used (to reduce chance of import cycle).

Reply:
Great !